### PR TITLE
additional sudo cmds needed for non-root user

### PR DESCRIPTION
**Pull request by mlammon**
Signed-off-by: Mike Lammon <mike.lammon@gmail.com>

### CODE REVIEW

File: get_images.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ if [ ! -f "$RHCOS_IMAGE_FILENAME" ]; then
     curl --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME}" "${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/${RHCOS_IMAGE_FILENAME}".gz
 fi
 
-mkdir -p "$IRONIC_DATA_DIR/html/images"
+sudo mkdir -p "$IRONIC_DATA_DIR/html/images"
+sudo chown $USER "$IRONIC_DATA_DIR/html/images"
```
Comment (yprokule) [inline get_images.sh]: Maybe change ownership for all the files within directory?

```suggestion
sudo chown -R $USER "$IRONIC_DATA_DIR/html/images"
```

**Comment by hardys** [comment]
Is this still needed since #199 landed?

I'm guessing not as `IRONIC_DATA_DIR="$WORKING_DIR/ironic"` and the user should own WORKING_DIR?

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/368/

**Comment by mlammon** [comment]
Thanks, good catch

**Pull request closed without merging**